src/store/reducers/tenant/tenant.ts

```diff
@@ -62,6 +71,19 @@ export const tenantApi = api.injectEndpoints({
             },
             providesTags: ['All'],
         }),
+        getClusterConfig: builder.query({
+            queryFn: async ({database}: {database?: string}, {signal}) => {
+                try {
+                    // FIXME!
```

Comment:
Please, leave comment what is needed to be fixed (e.g. "Pass database param when supported, remove filter"), otherwise it will be completely unclear what is the problem after some time